test/cloud_testing/platforms/centos8_x86_64_test.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+
+# source the common platform independent functionality and option parsing
+script_location=$(cd "$(dirname "$0")"; pwd)
+. ${script_location}/common_test.sh
+
+retval=0
+
+# allow apache access to the mounted server file system
+echo -n "setting SELinux labels for apache... "
+sudo chcon -Rv --type=httpd_sys_content_t /srv > /dev/null || die "fail"
+echo "done"
+
+# start apache
+echo -n "starting apache... "
+sudo systemctl start httpd > /dev/null 2>&1 || die "fail"
+echo "OK"
+
+# running unit test suite
+run_unittests --gtest_shuffle \
+              --gtest_death_test_use_fork || retval=1
+
+cd ${SOURCE_DIRECTORY}/test
+echo "running CernVM-FS client test cases..."
+CVMFS_TEST_CLASS_NAME=ClientIntegrationTests                                  \
+./run.sh $CLIENT_TEST_LOGFILE -o ${CLIENT_TEST_LOGFILE}${XUNIT_OUTPUT_SUFFIX} \
+                              -x src/005-asetup                               \
+                                 src/004-davinci                              \
+                                 src/007-testjobs                             \
+                                 src/056-lowspeedlimit                        \
+                                 src/084-premounted                           \
```

Comment:
Shouldn't this work, EL8 comes with fuse3 by default I thought.

Reply:
I haven't found any fuse3 files on the cc8 openstack image. The packages are available in the repository though.

Reply:
Ah, I see.  Then we need also two EL8 platforms for the two fuse versions